acceptance/windows-server-2012r2-fips/.acceptance/acceptance-cookbook/recipes/provision.rb

```diff
@@ -0,0 +1,11 @@
+execute "bundle exec ruby -e \"require 'mixlib/install'; puts Mixlib::Install.install_ps1, 'install'\" > ../.acceptance_data/powershell_install.ps1" do
```

Comment:
The acceptance tests have a few caveats in it's current state.  Rename `powershell_install.ps1`  on this line to something like `powershell_install_fips.ps1`. Otherwise it will overwrite the other script.